jujugui/static/gui/src/app/components/account/payment/payment.js

```diff
@@ -0,0 +1,215 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2017 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('account-payment', function() {
+
+  juju.components.AccountPayment = React.createClass({
+    displayName: 'AccountPayment',
+
+    propTypes: {
+      acl: React.PropTypes.object.isRequired,
+      addNotification: React.PropTypes.func.isRequired,
+      createPaymentMethod: React.PropTypes.func.isRequired,
+      createToken: React.PropTypes.func.isRequired,
+      createUser: React.PropTypes.func.isRequired,
```

Comment:
At the moment all the payment stuff is under a feature flag. So if the flag is turned off the methods don't exist to pass them down, but we only show these components when they're available.